dwbb/src/interface/DW_fp_i2flt.scala

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2024 Jiuyang Liu <liu@jiuyang.me>
+package org.chipsalliance.dwbb.interface.DW_fp_i2flt
+
+import chisel3._
+import chisel3.experimental.SerializableModuleParameter
+import upickle.default
+
+object Parameter {
+  implicit def rw: default.ReadWriter[Parameter] =
+    upickle.default.macroRW[Parameter]
+}
+
+case class Parameter(
+    sigWidth: Int = 23,
+    expWidth: Int = 8,
+    iSize: Int = 32,
+    iSign: Boolean = false
+) extends SerializableModuleParameter {
+  require(
+    Range.inclusive(2, 253).contains(sigWidth),
+    "Significand width must be between 2 and 253"
+  )
+  require(
+    Range.inclusive(3, 31).contains(expWidth),
+    "Exponent width must be between 3 and 31"
+  )
+  require(
+    Range.inclusive((3 + iSign), 512).contains(iSize),
```

Comment:
iSign is a Boolean?
```suggestion
    Range.inclusive((3 + iSign), 512).contains(iSize),
```

Reply:
`iSign` is a boolean but `iSize` is an integer.

Reply:
3 + iSign is Int + Boolean.
Please fix it.